blocks/codepen/codepen.css

```diff
@@ -0,0 +1,19 @@
+.cp_embed_wrapper { /* stylelint-disable-line selector-class-pattern */
```

Comment:
This is by convention per CodePen I assume? The classname?

Reply:
yep